src/utils/transactions/transactionClosePaymentErrorUtil.ts

```diff
@@ -0,0 +1,63 @@
+import { ViewOutcomeEnum } from "./TransactionResultUtil";
+
+export interface IClosePaymentErrorItem {
+  statusCode: string;
+  enablingDescriptions?: Array<string>;
+  outcome: ViewOutcomeEnum;
+}
+
+export const getClosePaymentErrorsMap = () => [
+  {
+    statusCode: "422",
+    enablingDescriptions: ["Node did not receive RPT yet"],
+    outcome: ViewOutcomeEnum.REFUND_IMMEDIATELY,
+  },
+  // this is out of scope, will be used later
+  /* {
+    statusCode: "422",
+    enablingDescriptions: ["Outcome already acquired"],
+    outcome: ViewOutcomeEnum.TAKING_CHARGE,
+  }, */
+  {
+    statusCode: "400",
+    enablingDescriptions: [
+      "Incorrect PSP-brokerPSP-Channel-Payment type configuration",
+      "Invalid additionalPaymentInformations",
+      "Invalid fee",
+      "Invalid idBrokerPSP",
+      "Invalid idChannel",
+      "Invalid idPSP",
+      "Invalid outcome",
+      "Invalid payment method",
+      "Invalid payment type",
+      "Invalid paymentMethod",
+      "Invalid paymentTokens",
+      "Invalid PSP/Canale for MBD",
+      "Invalid request",
+      "Invalid timestampOperation",
+      "Invalid token number",
+      "Invalid token",
+      "Invalid totalAmount",
+      "Invalid transactionId",
+      "Mismatched amount",
+      "Unacceptable outcome when token has expired",
+      "Wrong channel version",
+      "Wrong station version",
+    ],
+    outcome: ViewOutcomeEnum.REFUND_IMMEDIATELY,
+  },
+  {
+    statusCode: "404",
+    enablingDescriptions: [
+      "The indicated PSP does not exist",
+      "The indicated brokerPSP does not exist",
+      "The indicated channel does not exist",
+      "Unknown token",
+    ],
```

Comment:
For 400 and 404 you don't need to check error description because in all cases transactions are refunded.
Check this [PR](https://github.com/pagopa/pagopa-ecommerce-event-dispatcher-service/pull/356) for the backend implementation.

Reply:
I will need to ask clarification for this, because that PR you pointed is old and the whole point of this task is to refactor the usage of errors 4xx. You can read the specification [here](https://pagopa.atlassian.net/wiki/spaces/I/pages/1196130326/eCommerce+-+Messaggi+in+fase+closePayment)

Reply:
verified. done here 2163de1c7870f1fee562e0f4687a75a3fd108a90